### PR TITLE
Padding and alignment of benefit cards including footer and embedded cards

### LINKED ISSUE #1188: Increase benefit card padding to 35px
Design: https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b560156d4025ae7299f810f

### LINKED ISSUE #1240: Style benefit cards (colour, typography styles, icons, spacing, dropshadow)
https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b3e525390e27e2f31bf67d4

### PR DESCRIPTION
**Pull request by smcmurtry**
Closes #1188 
Closes #1240

### CODE REVIEW

**Comment by michellengaiCDS** [comment]
looks good to me!

**Review by michellengaiCDS — APPROVED** [review]

**Comment by smcmurtry** [comment]
Thanks @michellengaiCDS! It looks like we still need to fix some of your github permissions so that your approval will let me merge.

File: components/card_footer.js [hunk 1]
```diff
@@ -165,7 +171,7 @@ export class CardFooter extends Component {
           </ExpansionPanelSummary>
 
           <ExpansionPanelDetails timeout="auto" className={collapse}>
-            <Grid item xs={12}>
+            <div>
```
Comment (sastels) [inline components/card_footer.js:174]: Good call

**Review by sastels — APPROVED** [review]
🦑

**Pull request merged**